pig/resources/org/partiql/pig/templates/kotlin-folding-visitor.ftl

```diff
@@ -0,0 +1,84 @@
+[#ftl output_format="plainText"]
```

Comment:
This is unfortunately a copy & paste of the `kotlin-inspecting-visitor.ftl` file.  

The folding and inspecting visitors are very similar but different enough that it would have been a pain to try to make one FreeMarker `[#macro]` generate both.

Reply:
Fine by me.